sk0-front/src/app/auth/register/register.page.ts

```diff
@@ -59,6 +64,14 @@ export class RegisterPage {
       );
       return;
     }
+
+    if (this.registerForm.get("email")?.invalid) {
```

Comment:
We have this line or field which is similar (`this.registerForm.get("email")`, we probably might wanted to simplify this part.

Reply:
I dont really see how. If I say at line 57 that
const email = this.registerForm.get("email");
then I can rewrite line 68 in a simpler form (also line 61), but there will be a problem for the store.dispatch command, as it expects a string type, and email.value would be an AbstractControl<string | null, string | null> type.
So, from there I would have to rewrite the email's type in the store action, effect and reducer.

I dont really know how  I could resolve 'const email = this.registerForm.get("email")' into a string, as email?.value doesnt seem to work in the store registerStarted dispatch...

Reply:
I meant something like this, you do not need to use this.

```ts
const emailControl = this.registerForm.get("email");
const email = emailControl?.value;
const password = this.registerForm.get("password")?.value;
const name = this.registerForm.get("name")?.value;

if (!email || !password || !name) {
  await this.notificationService.alertError(
    "Kérlek tölts ki minden mezőt!"
  );
  return;
}

if (emailControl?.invalid) {
  await this.notificationService.alertError(
    "Kérlek adj meg egy érvényes email címet!"
  );
  return;
}

```

Reply:
Okay, got it. Fixed.